interface/ceed.c

```diff
@@ -436,6 +436,68 @@ int CeedSetDelegate(Ceed ceed, Ceed *delegate) {
   return 0;
 }
 
+/**
+  @brief Retrieve a delegate CEED for a specific object type
+
+  @param ceed           Ceed to retrieve delegate of
+  @param[out] delegate  Address to save the delegate to
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Developer
+**/
+int CeedGetObjectDelegate(Ceed ceed, Ceed *delegate, const char *objname) {
+  CeedInt ierr;
+
+  // Check for object delegate
+  for (CeedInt i=0; i<ceed->objdelegatecount; i++) {
+    if (!strcmp(objname, ceed->objdelegates->objname)) {
+      *delegate = ceed->objdelegates->delegate;
+      return 0;
+    }
+  }
+
+  // Use default delegate if no object delegate
+  ierr = CeedGetDelegate(ceed, delegate); CeedChk(ierr);
+ 
+  return 0;
+}
+
+/**
+  @brief Set a delegate CEED for a specific object type
+
+  @param ceed           Ceed to set delegate of
+  @param[out] delegate  Address to set the delegate to
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+int CeedSetObjectDelegate(Ceed ceed, Ceed *delegate, const char *objname) {
```

Comment:
This function only ever uses `*delegate` so presumably it can accept `Ceed delegate` instead.

Reply:
Fixed, and for `CeedSetDelegate` as well.